docs/server/source/data-models/inputs-outputs.rst

```diff
@@ -127,3 +127,26 @@ If there is only one *current owner*, the fulfillment will be a simple signature
 
 
 See the reference on :ref:`inputs <Input>` for descriptions of the meaning of each field.
+
+Multiple Current Owners
+```````````````````````
+
+If there are multiple *current owners*, the fulfillment will be a little different from `One Current Owner`_ (suppose it has two current owners).
```

Comment:
I would move the sentence inside the parentheses to be a stand-alone sentence, i.e.
```text
If there are multiple *current owners*, the fulfillment will be a little different from `One Current Owner`_. Suppose it has two current owners.
```